s3-cloudfront-spa/example-pattern.json

```diff
@@ -0,0 +1,63 @@
+{
+  "title": "AWS CloudFront and Amazon S3 for Single Page Application Hosting",
+  "description": "This project shows how to host a Single-Page Application (SPA) using AWS CloudFront & S3. CloudFront serves the SPA content from an S3 bucket configured for static hosting.",
+  "level": "200",
+  "framework": "Terraform",
+  "introBox": {
+    "headline": "How it works",
+    "text": [
+      "Here's how hosting a static website using Amazon CloudFront and Amazon S3 works:",
+      "- An Amazon S3 bucket is created and configured for static website hosting",
+      "- A CloudFront web distribution is created, with the S3 bucket configured as the origin server. CloudFront acts as a content delivery network (CDN) for the website",
+      "- When a user requests a file from the website, CloudFront first checks if the file is cached at one of its edge locations (data centers) closest to the user. If the file is cached, CloudFront serves it directly to the user.",
+      "- If the requested file is not cached at an edge location, CloudFront fetches the file from the S3 bucket origin. It then caches the file at the edge location for subsequent requests",
+      "- For subsequent requests, CloudFront serves the cached files from the nearest edge location to the user, providing low latency and fast content delivery.",
+      "- When website files are updated in the S3 bucket, CloudFront automatically detects the changes and retrieves the updated files from the origin. Cache invalidation can also be manually triggered to force CloudFront to fetch the latest versions."
+    ]
+  },
+  "gitHub": {
+    "template": {
+      "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/s3-cloudfront-spa",
+      "templateURL": "serverless-patterns/s3-cloudfront-spa",
+      "projectFolder": "s3-cloudfront-spa",
+      "templateFile": "template.tf"
+    }
+  },
+  "resources": {
+    "bullets": [
+      {
+        "text": "Amazon S3 bucket to host the website",
+        "link": "https://aws.amazon.com/s3/"
```

Comment:
Please use a more specific link

Reply:
I have updated the link to amazon s3 static hosting